modules/advanced-node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id="advanced-node-tuning-hosted-cluster_{context}"]
+= Advanced node tuning for hosted clusters by setting kernel boot parameters
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+For more advanced tuning in hosted control planes, which requires setting kernel boot parameters, you can also use the Node Tuning Operator. The following example shows how you can create a node pool with huge pages reserved.
+
+.Procedure
+
+. Create a config map that contains a tuned object manifest for creating 10 huge pages that are 2 MB in size. Save this config map manifest in a file named `tuned-hugepages.yaml`:
++
+[source,yaml]
+----
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-hugepages
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: hugepages
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Boot time configuration for hugepages
+              include=openshift-node
+              [bootloader]
+              cmdline_openshift_node_hugepages=hugepagesz=2M hugepages=50
+            name: openshift-node-hugepages
+          recommend:
+          - priority: 20
+            profile: openshift-node-hugepages
+----
++
+[NOTE]
+====
+The `.spec.recommend.match` field is intentionally left blank. In this case, this tuned is applied to all nodes in the node pool where this config map is referenced. Group nodes with the same hardware configuration into the same node pool. Otherwise, TuneD operands can calculate conflicting kernel parameters for two or more nodes that share the same node pool.
```

Comment:
```suggestion
The `.spec.recommend.match` field is intentionally left blank. In this case, this `Tuned` is applied to all nodes in the node pool where this `ConfigMap` is referenced. Group nodes with the same hardware configuration into the same node pool. Otherwise, TuneD operands can calculate conflicting kernel parameters for two or more nodes that share the same node pool.
```